src/common/constants.ts

```diff
@@ -174,6 +174,8 @@ export class Constants {
     public static openSampleEvent = "openSample";
     public static openSampleUrlEvent = "openSampleUrl";
 
+    public static isInternalPropertyName = "IsInternal";

```

Comment:
is the Uppercase of the first letter is a tradition in telemetry property?

Reply:
I followed the VS and live share extension rule. Will update the case to align with our own style